documentation/docs/25-build-and-deploy/90-adapter-vercel.md

```diff
@@ -64,6 +64,18 @@ And the following option apply to serverless functions:
 
 If your functions need to access data in a specific region, it's recommended that they be deployed in the same region (or close to it) for optimal performance.
 
+## Image Optimization
+
+You may set the `images` config to control how Vercel builds your images. See the [image configuration reference](https://vercel.com/docs/build-output-api/v3/configuration#images) for full details. As an example, you may set:
```

Comment:
One of the concerns in the linked issue was that folks also need to know how to consume it after setting these options. I wonder if linking to `@unpic/svelte` here would be helpful
```suggestion
You may set the `images` config to control how Vercel builds your images. This may be helpful when using image libraries such as [`@unpic/svelte`](https://unpic.pics/img/svelte/), which have built-in support for Vercel's image functionality. See the [image configuration reference](https://vercel.com/docs/build-output-api/v3/configuration#images) for full details. As an example, you may set:
```

Reply:
I lean towards having something built-in to the Vercel adapter instead, basically a small method you import from `@sveltejs/adapter-vercel` that takes care of it. AFAIK unpic loads the code for all the CDNs, it's not really treeshakeable.

Reply:
Great point. I filed an issue with unpic and will see if Matt responds. If we can fix it there that might be the best solution. I'll close this suggestion